### PR TITLE
User following

### PR DESCRIPTION
**Pull request by hunguyen1702**
---
This change is [Reviewable](https://reviewable.io/reviews/framgia/fels_nvh/23)

### CODE REVIEW

**Comment by hunguyen1702** [comment]
ready

**Comment by hungnh103** [comment]
conflicted

**Comment by hunguyen1702** [comment]
ready

File: app/controllers/relationships_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find params[:followed_id]
```
Comment (huongnguyenmta) [inline app/controllers/relationships_controller.rb]: dùng `find_by` method

File: app/controllers/relationships_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    current_user.follow @user
+    respond_to do |format|
+      format.html{redirect_to @user}
+      format.js
+    end
+  end
+
+  def destroy
+    @user = Relationship.find(params[:id]).followed
```
Comment (huongnguyenmta) [inline app/controllers/relationships_controller.rb]: dùng `find_by` method

File: config/routes.rb [hunk 3]
```diff
@@ -6,12 +6,17 @@
...
+    member do
+      get :following, :followers
+    end
+  end
   resources :categories, only: :index
   resources :words, only: %i(index show)
   resources :lessons, except: %i(new edit)
   resources :account_activations, only: :edit
   resources :password_resets, except: %i(destroy show index)
+  resources :relationships, only: [:create, :destroy]
```
Comment (huongnguyenmta) [inline config/routes.rb]: dùng `%i`

**Comment by huongnguyenmta** [comment]
commented

**Comment by hunguyen1702** [comment]
ready

**Pull request merged**